sql/head.sql

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Does pgspot pass on this? Or maybe run it on the final generated SQL.

Reply:
good call. let me try that.

Reply:
head.sql passes.

pgspot doesn't know what to do with plpython3u.
`Unknown function language: plpython3u`

Reply:
migration.sql [now passes](https://github.com/timescale/pgai/pull/47/commits/1361f5e88def739050d7b5df461774584cfe09df)